test/PowerShellEditorServices.Test/Console/InputPromptHandlerTests.cs

```diff
@@ -46,17 +47,17 @@ public void InputPromptHandlerReturnsValuesOfCorrectType()
                     CancellationToken.None);
 
             Assert.Equal(NameField, inputPromptHandler.LastField.Name);
-            inputPromptHandler.ReturnInputString(NameValue);
+            await inputPromptHandler.ReturnInputString(NameValue);
 
             Assert.Equal(AgeField, inputPromptHandler.LastField.Name);
-            inputPromptHandler.ReturnInputString(AgeValue.ToString());
+            await inputPromptHandler.ReturnInputString(AgeValue.ToString());
 
             Assert.Equal(BooksField + "[0]", inputPromptHandler.LastField.Name);
-            inputPromptHandler.ReturnInputString((string)BookItems[0]);
+            await inputPromptHandler.ReturnInputString((string)BookItems[0]);
             Assert.Equal(BooksField + "[1]", inputPromptHandler.LastField.Name);
-            inputPromptHandler.ReturnInputString((string)BookItems[1]);
+            await inputPromptHandler.ReturnInputString((string)BookItems[1]);
             Assert.Equal(BooksField + "[2]", inputPromptHandler.LastField.Name);
-            inputPromptHandler.ReturnInputString("");
+            await inputPromptHandler.ReturnInputString("");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4993607)